crates/typst-kit/src/package.rs

```diff
@@ -87,13 +87,9 @@ impl PackageStorage {
     ) -> PackageResult<PathBuf> {
         let subdir = format!("{}/{}/{}", spec.namespace, spec.name, spec.version);
 
-        if let Some(packages_dir) = &self.package_path {
-            let dir = packages_dir.join(&subdir);
-            if dir.exists() {
-                return Ok(dir);
-            }
-        }
-
+        // QUESTION: is it sometimes the case that both package_path
+        // and package_cache_path are Some? Is it a problem if we change
```

Comment:
A user may want to override a package from the preview namespace (which would land in the cache dir) with their own local version (in the package dir). The order should remain as is to support this use case.